### PR TITLE
[feat-5470]: Order incident table with headers instead of dropdown

### PR DESCRIPTION
**Pull request by OscarBakker**
Order incident table with headers instead of dropdown

Put sort icon in own folder
Extract Th component to own component
Styling table header
Add Chevron svg instead of string value

Ticket: [SIG-5470](https://gemeente-amsterdam.atlassian.net/browse/SIG-5470)

## Signalen

Before opening a pull request, please ensure:

- Make sure your PR title follows naming conventions: [feat-1234]: name feature
- Double-check your branch is based on `main` and targets `main`
- Pull request has tests (we are going for 100% coverage!)
- Code is well-commented, linted and follows project conventions
- Committed source code is headed by the correct SPDX license expression

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

[SIG-5470]: https://gemeente-amsterdam.atlassian.net/browse/SIG-5470?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/signals/incident-management/containers/IncidentOverviewPage/components/Th/Th.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  ordering?: SortOptions
+  changeOrder: (column: SortOptions) => void
+  sortOption: SortOptions
+  headerText: string
+  StyledComponent: any
+}) => {
+  return (
+    <StyledComponent
+      $underline={
+        ordering && ordering.replace('-', '') === sortOption.replace('-', '')
```
Comment (vdegraaf) [inline src/signals/incident-management/containers/IncidentOverviewPage/components/Th/Th.tsx:22]: This funtionality also exists in Sort Icon. Maybe good to have a property or function that identifies the active sort?

File: src/signals/incident-management/containers/IncidentOverviewPage/index.js
Comment (vdegraaf) [inline src/signals/incident-management/containers/IncidentOverviewPage/index.js:1]: Would've been a good opportunity to refactor this to TypeScript.

**Review by vdegraaf — COMMENTED** [review]
Works nice! Some comments:
- Do we want to release this before  https://gemeente-amsterdam.atlassian.net/browse/SIG-5482 is done? Since it doesn't work properly when you've search via the search bar. 
- when active the header is underlined, why? Since I couldn't find it in the design. 
- The icons are changing abrupt. Better to change it to css animation like on the new filter page or category detail where you change the icon.

File: src/signals/incident-management/containers/IncidentOverviewPage/index.js
Comment (OscarBakker) [reply src/signals/incident-management/containers/IncidentOverviewPage/index.js:1]: If I finish the other ticket ;)

**Comment by OscarBakker** [comment]
> Works nice! Some comments:
> 
> * Do we want to release this before  https://gemeente-amsterdam.atlassian.net/browse/SIG-5482 is done? Since it doesn't work properly when you've search via the search bar.
> * when active the header is underlined, why? Since I couldn't find it in the design.
> * The icons are changing abrupt. Better to change it to css animation like on the new filter page or category detail where you change the icon.
Screenshot 2023-11-15 at 10 39 52

**Review by vdegraaf — APPROVED** [review]

File: src/signals/incident-management/containers/IncidentOverviewPage/components/Th/Th.tsx [hunk 2]
```diff
@@ -18,10 +17,7 @@ const ThComponent = ({
   StyledComponent: any
```
Comment (vdegraaf) [inline src/signals/incident-management/containers/IncidentOverviewPage/components/Th/Th.tsx:17]: Should be typed

**Review by vdegraaf — APPROVED** [review]
LGTM
just small type to add

**Pull request merged**